changelog/9045.removal.md

```diff
@@ -0,0 +1,10 @@
+Setting `model_confidence=linear_norm` is deprecated and will be removed in Rasa Open Source `3.0.0` .
+
+Rasa Open Source `2.3.0` introduced `linear_norm` as a possible value for `model_confidence`
+parameter in machine learning components such as `DIETClassifier`, `ResponseSelector` and `TEDPolicy`.
+Based on user feedback, we have identified multiple problems with this option and hence
+using `model_confidence=linear_norm` is now deprecated and
```

Comment:
Rephrasing:
"Based on user feedback, we have identified multiple problems with this option. Therefore,`model_confidence=linear_norm` is now deprecated and"